map_arm/include/map_arm/MapArm.hpp

```diff
@@ -0,0 +1,80 @@
+// Copyright 2021 Aron Svastits
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef MAP_ARM__MAPARM_HPP_
+#define MAP_ARM__MAPARM_HPP_
+
+#include <Eigen/Geometry>
+
+#include <cinttypes>
+#include <vector>
+#include <string>
+#include <map>
+#include <memory>
+#include <cmath>
+
+#include "rclcpp/rclcpp.hpp"
+#include "sensor_msgs/msg/joint_state.hpp"
+#include "visualization_msgs/msg/marker_array.hpp"
+#include "lifecycle_msgs/srv/change_state.hpp"
+#include "std_srvs/srv/trigger.hpp"
+#include "std_srvs/srv/set_bool.hpp"
+#include "utils/GeometryUtils.hpp"
+
+
+namespace filter_points
+{
+
+// TODO(kovacsge11) maybe derive from ROS2BaseNode
```

Comment:
Writing this comment so that you notice this TODO, this is a possible not so high prio TODO.